app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -57,8 +64,14 @@ def process_event(event)
     when 'Node'
       event_data[:container_node_name] = event_data[:name]
     when 'Pod'
+      /^spec.containers{(?<container_name>.*)}$/ =~ event_data[:fieldpath]
+      if container_name.nil?
+        event_data[:container_group_name] = event_data[:name]
+      else
+        event_data[:kind] = 'Container'
+        event_data[:container_name] = container_name
+      end
```

Comment:
Now that I think of it we should set `event_data[:container_group_name]` regardless.

      ...
    when 'Pod'
      /^spec.containers{(?<container_name>.*)}$/ =~ event_data[:fieldpath]
      event_data[:container_name] = container_name unless container_name.nil?
      event_data[:container_group_name] = event_data[:name]
    end